### PR TITLE
Add download button functionality

### PR DESCRIPTION
**Pull request by fionaphillips**
Resolves [CG-47](https://unosquarebelfast.atlassian.net/secure/RapidBoard.jspa?rapidView=12&projectKey=CG&modal=detail&selectedIssue=CG-47)

Dependency
- [x] https://github.com/UnosquareBelfast/CVGenerator-frontend/pull/26

### CODE REVIEW

File: src/components/ModalButtons/DownloadButton/styled.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 
-const StyledDownloadButton = styled(Button)`
+const StyledDownloadButton = styled.a`
   width: 90%;
   margin: 25px 0 0 0;
+  text-align: center;
+  text-decoration: none;
+  background: ${({ theme }) => theme.colors.buttonBlue};
+  border-radius: 5px;
+  color: white;
```
Comment (girwin90) [inline src/components/ModalButtons/DownloadButton/styled.js]: Do you have a theme variable for white?

File: src/components/CVModal/CVModalButtonBar/index.js [hunk 2]
```diff
@@ -3,15 +3,16 @@ import { PropTypes as PT } from 'prop-types';
 import { CancelModalButton, DownloadModalButton } from 'Components';
 import StyledButtonBar from './styled';
 
-export const CVModalButtonBar = ({ handleCancelClick }) => (
+export const CVModalButtonBar = ({ handleCancelClick, cvData }) => (
```
Comment (girwin90) [inline src/components/CVModal/CVModalButtonBar/index.js]: I'm seeing `cvData` used a lot. It's used for a href and passed through a few areas. What does it contain?

Comment (fionaphillips) [reply src/components/CVModal/CVModalButtonBar/index.js]: It contains the endpoint for downloading the employee CV. I've changed it to cvURL which hopefully is more descriptive.

Comment (girwin90) [reply src/components/CVModal/CVModalButtonBar/index.js]: Perfect.

**Review by girwin90 — APPROVED** [review]

**Review by BrownMark — APPROVED** [review]
LGTM :shipit:

**Pull request merged**